src/ast/mod.rs

```diff
@@ -236,6 +236,10 @@ pub enum Expr {
         expr: Box<Expr>,
         key: Ident,
     },
+    /// `IS FALSE` operator
```

Comment:
It is somewhat unfortunate we don't have `Expr::Is(Box<Expr>, Box<Expr>)` which would handle all these cases.

What do you think @ovr ? Should I file a ticket to try and reduce the duplication in `Expr`?

Reply:
IMHO: It's a breaking change, which doesn't give any advantages over this. Right now, it's really easy to implement a match pattern over this because names are really helpful.